spec/function.dd

```diff
@@ -2354,11 +2355,11 @@ int foo(char[] s)
 const int x = foo("1");
 ---
 
-        $(P is illegal, because the runtime code for foo() cannot be
+        $(P is illegal, because the runtime code for `foo` cannot be
```

Comment:
Slightly off topic, but if possible, we should combine this paragraph with one above as currently they're rendered as two independent paragraphs, while they semantically form one single paragraph with an example in the middle.